### PR TITLE
[INVE-14357] - Localising a type variable to prevent it from polluting the globalScope

### PR DESCRIPTION
**Pull request by corrigancd**

### CODE REVIEW

File: src/ext/GeometryUtil.js [hunk 1]
```diff
@@ -70,7 +70,7 @@
 
 			if (isMetric) {
 				units = ['ha', 'm'];
-				type = typeof isMetric;
+				const type = typeof isMetric;
```
Comment (corrigancd) [inline src/ext/GeometryUtil.js:73]: @szydan, window.type doesn't get assigned the `boolean` when this type variable is localised. The only place this type variable seems to be used is within this if statement and everything we need to use works on manual check. The other files in this PR (other than the version change) are relating to the build script. When this is merged, I'll verify it works via `npm i` on kibi-internal and will make a sibling PR there.

Comment (szydan) [reply src/ext/GeometryUtil.js:73]: Ahah here is our scope leak 
Nice ;-)

**Review by szydan — APPROVED** [review]

**Comment by szydan** [comment]
@corrigancd is this leak present in the original leaflet draw plugin repo ?
If yes please open an issue + fixing PR there

**Comment by corrigancd** [comment]
It is present on the original repo. I'll make a PR on the base repo, although nothing has been reviewed/change on it in 3~ years

**Pull request merged**